components/constellation/constellation.rs

```diff
@@ -226,7 +226,7 @@ pub struct Constellation<Message, LTF, STF> {
 
     /// A channel for the constellation to send messages to the
     /// timer thread.
-    scheduler_chan: IpcSender<TimerSchedulerMsg>,
+    scheduler_chan: Sender<TimerSchedulerMsg>,
```

Comment:
There is no ```Constellation::new()```, do you mean ```Constellation::start()```?

And I'm a little confused because ```Constellation::start()``` is the only place that creates this ```Constellation``` struct (the only place where this field is being set).  Do you mean that there is a second one being created elsewhere?  If so, should one of them go away, or did you just mean that this comment should be updated?

Reply:
When multiprocess mode is enabled, there is no need a scheduler in the process where the Constellation runs. When multiprocess mode is not enabled, there should be one scheduler created.